docs/migration/guide_aspnet.md

```diff
@@ -0,0 +1,102 @@
+# Guide for Asp.Net applications
+
+## Intro
+
+The following guide details how to remove Application Insights from an ASP.NET web app and how to set up the OpenTelemetry SDK.
+
+## Prerequisites
+
+- An ASP.NET web application already instrumented with Application Insights
+- A actively supported version of .NET (link)
+
+## Steps to Migrate
+
+### Step 1: Remove Application Insights SDK
+
+When you first added ApplicationInsights to your project, the SDK would have added a config file and made some edits to the web.config.
```

Comment:
https://learn.microsoft.com/en-us/azure/azure-monitor/app/asp-net#how-can-i-uninstall-the-sdk